roles/installer/defaults/main.yml

```diff
@@ -83,7 +99,27 @@ node_selector: ''
 #         app.kubernetes.io/name: "<resourcename>"
 topology_spread_constraints: ''
 
-affinity: {}
+# Add a topologySpreadConstraints for the task pods.
+# Specify as literal block. E.g.:
+# task_topology_spread_constraints: |
+#   - maxSkew: 100
+#     topologyKey: "topology.kubernetes.io/zone"
+#     whenUnsatisfiable: "ScheduleAnyway"
+#     labelSelector:
+#       matchLabels:
+#         app.kubernetes.io/name: "<resourcename>""
```

Comment:
```suggestion
#         app.kubernetes.io/name: "<resourcename>""
```
```
#         app.kubernetes.io/name: "<resourcename>"
```

Reply:
Resolved in https://github.com/ansible/awx-operator/pull/1311